src/utils/redirectless.js

```diff
@@ -3,16 +3,15 @@ const allowedAuthRequestParameters = ['client_id', 'response_type', 'code_challe
   'redirect_uri', 'scope', 'state', 'idp', 'pfidpadapterid', 'access_token_manager_id', 'aud', 'nonce', 'prompt',
   'acr_values', 'max_age', 'login_hint', 'ui_locales', 'id_token_hint', 'claims_locales'];
 
-export const INTERACT_WITH_USER_AUTHZ = 'userAuthz';
+export const FLOW_TYPE_USER_AUTHZ = 1;
```

Comment:
Should we define a constant like `FLOW_TYPE_AUTHZ = 0` as well, for completeness?

Reply:
I'm not sure it would be beneficial at this point. The redirectless.js file is definitely the best place to define it too. 🤔

Reply:
I'm sort of thinking of this similar to a Java enum. Even we don't use it anywhere, defining `FLOW_TYPE_AUTHZ = 0` can help developers understand what the different possibilities are.

Reply:
done